docs/source/pages/using.rst

```diff
@@ -19,29 +19,29 @@ To run Macaron on a Github public repository, we use the following command:
 
 .. code-block:: shell
 
-  ./run_macaron.sh analyze -rp <repo_path>
+  ./run_macaron.sh macaron analyze -rp <repo_path>
```

Comment:
This file still contains `./run_macaron.sh macaron` command. Please fix these.

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/269/commits/602d68376381eb591484d8df2ecaee0e26dd0e7e